app/Http/Controllers/Admin/UserController.php

```diff
@@ -14,7 +16,23 @@ class UserController extends Controller
      */
     public function index()
     {
-        return view('backend.users.index');
+        $fields = [
+            'users.id',
+            'users.employ_code',
+            'users.name',
+            'users.team',
+            'users.email',
+            'users.is_admin',
+            'users.avatar_url',
+            DB::raw('COUNT(DISTINCT(borrows.id)) AS total_borrowed'),
```

Comment:
em thử câu quẻy này thử:

> $fields = [
            'users.id',
            'users.employ_code',
            'users.name',
            'users.team',
            'users.email',
            'users.is_admin',
            'users.avatar_url',
        ];
        $users = User::select($fields)
        ->withCount(['books', 'borrows'])
        ->orderBy('id')
        ->paginate(config('define.users.row_count'));

Update cái quan hệ `books` trong model `users`

> public function books()
    {
        return $this->hasMany(Book::class, 'from_person', 'employ_code');
    }